.github/workflows/ci.yml

```diff
@@ -0,0 +1,72 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - master
+  pull_request: {}
+
+jobs:
+  test:
+    name: "Tests"
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14.x
+      - name: Install Dependencies
+        run: npm install --frozen-lockfile
+      # - name: Lint
+      #  run: npm run lint
+      - name: Run Tests
+        run: yarn test
+
+  floating:
+    name: "Floating Dependencies"
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14.x
```

Comment:
I think we are still supporting v12 in the ecosystem generally, esp since we have it in the package.json

Reply:
I just copied this file from another project without putting a lot of thought into it. I've changed it to just use the blueprint which seems more inline with the other adopted addons.